ferdinand-porsche-fern-fachhochschule.csl

```diff
@@ -166,13 +170,17 @@
         <else-if type="chapter paper-conference" match="any">
           <text macro="title" prefix=" " suffix="."/>
           <group prefix=" " delimiter=" ">
-            <text term="in" text-case="capitalize-first"/>
-            <text macro="editor"/>
-            <text variable="container-title" font-style="italic" suffix="."/>
+            <text term="presented at" text-case="capitalize-first"/>
```

Comment:
cool -- aber mir ist immer nocht nicht klar ob das richtig ist: die Aenderung hier betrifft ja auch Kapitel ("Beitraege in Sammelwerken" im verlinkten Leitfaden) und dort soll defnitive "In" und nicht "Vortrag auf der..." verwendet werden.